ui-speedspacechart/src/types/chartTypes.ts

```diff
@@ -44,6 +47,13 @@ export type TrainDetails = {
   marecoSpeedText: string;
   effortText: string;
   electricalProfileText: string;
+  powerRestrictionText: string;
   previousGradientText: number;
   modeText: string;
 };
+
+export enum LinearLayerHeights {
```

Comment:
Hm, I find it a bit confusing to make this an `enum`, should this be an object instead (like `layerHeights` was)?

Reply:
Done